Documentation/codereview-policy.md

```diff
@@ -0,0 +1,45 @@
+### Code Review Policy
+
+#### How to review pull requests  
+
+This is a guide for core developers and contributors who want to help with code reviewing PRs on the fullnode.  
+We welcome anyone to help with code reviews, from typos to suggestions in design.   
+
+#### Why are code reviews important?  
+It's essential that PRs are properly code-reviewed.   
+Changes to the code, even small changes as incrementing an integer, can result in critical consensus bugs and loss of funds if not reviewed properly.  
+Some PRs are more severe than others in importance (for example changes to tests may not require the same level of checks as changes to the consensus code).  
+For that reason we have a PR review policy with 3 tags. A PR must be associated with a PR review policy.
+
+#### Requirements
+
+The requirement for each PR policy
+
+*pr-review-critical*
+- Code must be reviewed by at least two core devs
+- All tests must pass (Unit-test and Integration-tests) (run by auther and the two reviewers)
+- All 4 chains (Bitcoin main/test, Stratis main/test) need to have been synced (to an agreed height) (run by auther and 2 reviewers)
```

Comment:
`either`

And we should probably specify the heights exactly.

Reply:
I was thinking that could be discussed in the review itself

Reply:
It would just distract both the coder and the reviewer, so I think it's better to just define it once in the policy.

Reply:
>It would just distract both the coder and the reviewer, so I think it's better to just define it once in the policy.

I agree. 

I think for stratis main and test syncing till the tip is required. 
For BTC test and main- hard to tell- potentially we want to sync until the last checkpoint or the last BIP activation but that still might take a very long time.

Reply:
My take is If you find this to be an issue we can revisit